src/Searcher.cpp

```diff
@@ -125,9 +131,13 @@ std::vector<Node> Searcher::expand(const Node& parent,
     return children;
 }
 
-void Searcher::move_order(std::vector<Node>& nodes) {
-    // Do nothing for now
-    // TODO: Better move ordering
+void Searcher::move_order(Who team) {
+    for (auto&& moves : ordered_moves) {
+        if (team == Who::HOME)
+            std::sort(moves.second.begin(), moves.second.end(), std::greater<Node>());
+        else
+            std::sort(moves.second.begin(), moves.second.end());
```

Comment:
```
auto comp = team == Who::HOME ? std::greater<Node>() : std::less<Node>();
```

would probably look better, and would make me not have to point out about using braces even for single-line if statements :grimacing: